src/fips.c

```diff
@@ -379,6 +380,7 @@ static int load_known_hmac(const char *path, unsigned char **hmac, long *hmaclen
 	rc = 0;
 end:
 	fclose(fp);
+	OPENSSL_cleanse(known_hmac_str, strlen(known_hmac_str));
```

Comment:
`known_hmac` and `known_hmac_str` are just read buffers, and are not involved in any hash calculation. Strictly speaking it should not be seccessary to zerorize them as well. But on the other side, it won't hurt to clean them as well. So I would leave it as is.